From d6f7ebcf32d6317e95bfa67f5954e0269f1d11ab Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Wed, 3 Oct 2012 13:33:56 +0200
Subject: [PATCH 481/628] usb: ehci: Do not use function name in printk

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 467678f7f2d9bc74b250ba90871e95decc83b8bb

Use __func__ instead.

Signed-off-by: Michal Simek <monstr@monstr.eu>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/usb/host/ehci-xilinx-usbps.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/host/ehci-xilinx-usbps.c b/drivers/usb/host/ehci-xilinx-usbps.c
index 9c96b85..e3262f8 100755
--- a/drivers/usb/host/ehci-xilinx-usbps.c
+++ b/drivers/usb/host/ehci-xilinx-usbps.c
@@ -169,7 +169,7 @@ static int usb_hcd_xusbps_probe(const struct hc_driver *driver,
 	ehci = hcd_to_ehci(hcd);
 	if (pdata->otg) {
 #ifdef CONFIG_XILINX_ZED_USB_OTG
-		pr_info ("usb_hcd_xusbps_probe: Have OTG assigned.\n");
+		pr_info ("%s: Have OTG assigned.\n", __func__);
 
 		retval = otg_init(pdata->otg);
 		if (retval) {
@@ -190,7 +190,7 @@ static int usb_hcd_xusbps_probe(const struct hc_driver *driver,
 		xusbps_update_transceiver();
 	} else {
 #ifdef CONFIG_XILINX_ZED_USB_OTG
-		pr_info ("usb_hcd_xusbps_probe: No OTG assigned!\n");
+		pr_info ("%s: No OTG assigned!\n", __func__);
 		if (!pdata->otg) {
 			pdata->otg = otg_ulpi_create (&ulpi_viewport_access_ops,
 				ULPI_OTG_DRVVBUS | ULPI_OTG_DRVVBUS_EXT);
@@ -199,7 +199,7 @@ static int usb_hcd_xusbps_probe(const struct hc_driver *driver,
 				ehci->ulpi = pdata->otg;
 			}
 		}
-		pr_info ("usb_hcd_xusbps_probe: OTG now assigned!\n");
+		pr_info ("%s: OTG now assigned!\n", __func__);
 #endif
 
 		retval = usb_add_hcd(hcd, irq, IRQF_DISABLED | IRQF_SHARED);
-- 
1.7.5.4

